Services/Models/Simulation.cs

```diff
@@ -57,7 +57,7 @@ public class Simulation
         public IList<string> IotHubConnectionStrings
         {
             get => this.iotHubConnectionStrings;
-            set => this.iotHubConnectionStrings = value ?? new List<string>();
+            set => this.iotHubConnectionStrings = value ?? new List<string> { ServicesConfig.USE_DEFAULT_IOTHUB };
```

Comment:
Why is this needed here? I see that there is code that appears to be setting this in the API model, perhaps we should set the default there if we're getting to a point where the default isn't set properly in the service model.

Reply:
Setting default value here. We used to have a default value which changed with my changes